### DISCUSSION TITLE
Suggestion: Update to the loading animation

### DISCUSSION THREAD

**Discussion by shadowtajun** [torrinworx/Blend_My_NFTs#165] [Ideas]
Hey! amazing add-on. 
I'd like to suggest the following loading animation would look more like DNA. While still looking like a loading screen. Just as a fun aside
```
                " [==   --]",
                " [ == -- ]",
                " [  ==-  ]",
                " [  -==  ]",
                " [ -- == ]",
                " [--   ==]",
                " [ -- == ]",
                " [  --=  ]",
                " [  =--  ]",
                " [ == -- ]",
```
But what I'd **really love** is a way for the information of in/valid combinations to be displayed (optionally - maybe a checkbox in the UI before processing) so that we can see that the data is progressing and not just stalling when the logic gets super complicated 

(I've had successful computations take 10+ hours)

Is this at all possible?

**Comment by cjwmaya** [comment]
你好，我是也毛儿，信件已经收到，
